src/main/java/com/infoshareacademy/service/RecipeService.java

```diff
@@ -123,4 +123,8 @@ public void deleteRecipeById(Long id) {
         recipeRepository.deleteById(id);
     }
 
+    public void deleteAllRecipes(){
+        recipeRepository.deleteAll();
+    }
```

Comment:
Moim zdaniem metoda do przerobienia - nie chcemy usuwac wszystkich przepisow z bazy tylko wszystkie przepisy konkretnego uzytkownika. Wiem że to wymaga żeby bylo logowanie itp. ale jest metoda getUser w user service (poki co zwraca sztywno id 1) i mysle że trzeba by ja wykorzystać i powiazac usuwanie przepisów z uzytkownikiem.